static/app/views/issueDetails/updatedHeader.tsx

```diff
@@ -0,0 +1,367 @@
+              <AssigneeSelectorDropdown
+                group={group}
+                loading={assigneeLoading}
+                onAssign={(assignedActor: AssignableEntity | null) =>
+                  handleAssigneeChange(assignedActor)
+                }
+                onClear={() => handleAssigneeChange(null)}
+                trigger={(props, isOpen) => (
+                  <StyledDropdownButton
+                    {...props}
+                    borderless
+                    aria-label={t('Modify issue assignee')}
+                    size="zero"
+                  >
+                    <AssigneeBadge
+                      assignedTo={group.assignedTo ?? undefined}
+                      assignmentReason={
+                        group.owners?.find(owner => {
+                          const [_ownershipType, ownerId] = owner.owner.split(':');
+                          return ownerId === group.assignedTo?.id;
+                        })?.type
+                      }
+                      loading={assigneeLoading}
+                      chevronDirection={isOpen ? 'up' : 'down'}
+                    />
+                  </StyledDropdownButton>
+                )}
+              />
+            </Wrapper>
+            {participants.length > 0 && (
+              <Wrapper>
+                {t('Participants')}
+                <div>
+                  <StyledAvatarList
+                    users={userParticipants}
+                    teams={teamParticipants}
+                    avatarSize={18}
+                    maxVisibleAvatars={2}
+                    typeAvatars="participants"
+                  />
+                </div>
+              </Wrapper>
+            )}
+            {displayUsers.length > 0 && (
+              <Wrapper>
+                {t('Viewers')}
+                <StyledAvatarList
+                  users={displayUsers}
+                  avatarSize={18}
+                  maxVisibleAvatars={2}
+                />
+              </Wrapper>
+            )}
+          </PriorityAssignee>
+        </InfoWrapper>
+        <GroupHeaderTabs {...{baseUrl, disabledTabs, eventRoute, group, project}} />
+      </div>
+    </Layout.Header>
+  );
+}
+
+const TitleWrapper = styled('div')`
+  @media (min-width: ${p => p.theme.breakpoints.small}) {
+    max-width: 65%;
+  }
+`;
+
+const StyledEventOrGroupTitle = styled(EventOrGroupTitle)`
+  font-size: inherit;
+`;
+
+const StyledFeatureBadge = styled(FeatureBadge)`
+  align-items: flex-start;
+`;
+
+const TitleHeading = styled('div')`
+  display: flex;
+  line-height: 2;
+  gap: ${space(1)};
+`;
+
+const StyledBreak = styled('hr')`
+  margin-top: ${space(3)};
+  margin-bottom: 0;
+`;
+
+const MessageWrapper = styled('div')`
+  display: flex;
+  color: ${p => p.theme.gray300};
+  gap: ${space(1)};
+`;
+
+const InfoWrapper = styled('div')<{isResolved: boolean}>`
+  padding-top: ${space(1)};
+  padding-bottom: ${space(1)};
+  display: flex;
+  justify-content: space-between;
```

Comment:
We'll also need a gap to ensure they don't collide:
`gap: ${space(1)}` will match the button bar if you like

![image](https://github.com/user-attachments/assets/209cc3f5-7937-4291-aec8-928203907f85)